cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting to connect to database...")
+        db_connect = None
```

Comment:
Consider setting a maximum number of attempts or a timeout to prevent the script from potentially running indefinitely if the database is not available. This can help avoid resource exhaustion and make the script more robust.